### ISSUE TITLE
support `asdf.info` output for custom tags

### ISSUE THREAD

**Issue by CagtayFabry** [BAMWelDX/weldx#685]
implement `__asdf_traverse__` for costum weldx tags, see https://github.com/asdf-format/asdf/pull/1052

**Comment by marscher** [comment]
The traverse method needs to be implemented for the implementation classes, e.g. CSM and so forth, right?
Or is there a way to hide this asdf related detail somewhere in the asdf related code?

**Comment by CagtayFabry** [comment]
Yes that method is needed purely on the python side if I understand correctly

But the basic idea should also be helpful for https://github.com/BAMWelDX/weldx-widgets/issues/13, maybe we can combine these two in the future
